spacy_keyword_extraction_api/api_router.py

```diff
@@ -1,26 +1,147 @@
+from collections import Counter
 import logging
-from typing import Sequence
+from typing import Optional, Sequence
 from typing_extensions import TypedDict
 
-from fastapi import APIRouter
+from fastapi import APIRouter, Body
 
+from spacy_keyword_extraction_api.api_router_typing import (
+    BatchExtractKeywordsRequestTypedDict,
+    BatchKeywordsResponseTypedDict,
+    KeywordsRequestDataTypedDict,
+    KeywordsResponseDataTypedDict,
+    KeywordsResponseKeywordTypedDict,
+    KeywordsResponseMetaTypedDict
+)
 from spacy_keyword_extraction_api.extract_keywords import KeywordExtractor
 
 
 LOGGER = logging.getLogger(__name__)
 
 
+EXAMPLE_BATCH_EXTRACT_KEYWORDS_REQUEST: BatchExtractKeywordsRequestTypedDict = {
+    'data': [{
+        'type': 'extract-keyword-request',
+        'id': 'doc-1',
+        'attributes': {
+            'content': 'I am interested in biochemistry and neuroscience.'
+        },
+        'meta': {
+            'extra_id': 'doc-1/v1'
+        }
+    }]
+}
+
+EXAMPLE_BATCH_KEYWORDS_RESPONSE: BatchKeywordsResponseTypedDict = {
```

Comment:
Is the response structure different than before and also between two endpoints? How will we manage this?